### DISCUSSION TITLE
How can I customize the default spacing between components in shadcn to fit my design?

### DISCUSSION THREAD

**Discussion by 0xariii** [shadcn-ui/ui#6049] [Q&A] [answered]
I am using shadcn in my project and I want to tweak the spacing between components globally to align with my design. Is there a way to do this without overriding every component manually?

**Comment by georgeeburt** [comment, marked as answer]
You can customise the default spacing globally in shadcn by extending the tailwind CSS configuration file (`tailwind.config.js`) since shadcn is built on tailwind.

Here is a simple example:

```
module.exports = {
  theme: {
    extend: {
      spacing: {
        'custom-1': '4px',
        'custom-2': '8px',
        // Add more as needed
      },
    },
  },
};
```

Hope this helps.

**Reply by 0xariii** [reply]
thank you
